proto/viam/app/v1/app.proto

```diff
@@ -987,3 +991,31 @@ message CreateKeyResponse {
   string key = 1;
   string id = 2;
 }
+
+message DeleteKeyRequest {
+  string key = 1;
+  string id = 2;
+}
+
+message DeleteKeyResponse {}
+
+message AuthorizationDetails {
+  string authorization_type = 1;
+  string authorization_id = 2;
+  string resource_type = 3;
+  string resource_id = 4;
+  string org_id = 5;
+}
```

Comment:
[q] why did you decide to create a new message instead of reusing the existing `Authorization` message? was it to avoid the redundant fields (identity_id, identity_type, etc)?

Reply:
Yeah I thought it made more logical sense with how the information will likely be used. The additional duplicated information can be a bit confusing I think